test/unit/kv-extreme-scale.unit.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+const Connector = require('../../lib/kv-extreme-scale')._Connector;
+const dsFactory = require('../helpers/data-source-factory');
+const expect = require('../helpers/expect');
+
+describe('eXtreme Scale Connector', () => {
+  describe('mapname', () => {
+    const modelName = 'TestModel';
+    // connector defaults key expiration to LastUpdateTime with Optimistic
+    // locking
+    const keyExpirationDefaults = '.LUT.O';
+    let ds;
+    before(createDataSource);
+
+    context('without user-defined mapname', () => {
+      const expectedMapname = modelName + keyExpirationDefaults;
+      before(createTestModel);
```

Comment:
I find it puzzling how is `expectedName` related to the model created by `createTestModel`.

I am proposing to:
 - make `createTestModel` name more descriptive by describing what kind of model is created
 - or inline the function here, so that it's immediately obvious what kind of model is created.